CHANGELOG.md

```diff
@@ -2,15 +2,16 @@
 
 All notable changes to this project will be documented in this file, in reverse chronological order by release.
 
-## 0.2.1 - TBD
+## 0.3.0 - TBD
 
 ### Added
 
 - Nothing.
 
 ### Changed
 
-- Nothing
+- [#14](https://github.com/zendframework/zend-expressive-authentication/issues/14)
+  changed the `getUsername` with `getIdentity` in UserInterface.
```

Comment:
Change to:

```markdown
renames the method `UserInterface::getUsername()` to `UserInterface::getIdentity()`.
```

Depending on another suggested change below, we may also need to detail something about the return value change.